cookbooks/bcpc/recipes/grub.rb

```diff
@@ -0,0 +1,60 @@
+# vim: tabstop=2:shiftwidth=2:softtabstop=2
```

Comment:
These upstart code will probably be broken in 16.04 and 18.04 when we start to move. Let's add at least a "FIXME: 16.04" or something so we can begin to track code that should be updated.

Reply:
Updated.